src/index.rs

```diff
@@ -516,9 +471,41 @@ impl ResourceIndex {
         self.path2id.insert(path, entry);
     }
 
+    /// Removes the given resource ID from the index and returns an update
+    /// containing the deleted entries
+    pub fn forget_id(&mut self, old_id: ResourceId) -> Result<IndexUpdate> {
```

Comment:
Same as this? https://github.com/ARK-Builders/arklib/blob/index/update-one/src/index.rs#L393

Reply:
I didn't really change the function logic. Should I rename it to `track_deletion`?